bindings/go/src/_stacktester/stacktester.go

```diff
@@ -661,6 +662,23 @@ func (sm *StackMachine) processInst(idx int, inst tuple.Tuple) {
 			t = append(t, sm.waitAndPop().item)
 		}
 		sm.store(idx, []byte(t.Pack()))
+	case op == "TUPLE_PACK_VERSIONSTAMP":
+		var t tuple.Tuple
+		count := sm.waitAndPop().item.(int64)
+		for i := 0; i < int(count); i++ {
+			t = append(t, sm.waitAndPop().item)
+		}
+
+		incomplete, err := t.HasIncompleteVersionstamp()
```

Comment:
I think this should use the `err` result from `PackWithVersionstamp` rather than `HasIncompleteVersionstamp` to test that the packing itself should fail.

Reply:
👍